lib/deluge/rpc/version.rb

```diff
@@ -1,5 +1,5 @@
 module Deluge
   module Rpc
-    VERSION = "0.2.2"
+    VERSION = "0.2.3"
```

Comment:
Please don't change version by yourself. I'll update it right before gem is released